.golangci.yml

```diff
@@ -77,8 +77,6 @@ linters-settings:
     field-writes-are-uses: true
     # Treat IncDec statement (e.g. `i++` or `i--`) as both read and write operation instead of just wr◊ite. Default: false
     post-statements-are-reads: true
-    # Mark all exported identifiers as used. Default: true
```

Comment:
is this dropped since it no longer works with the newer version of the linter?

Reply:
Correct, `exported-is-used` is deprecated, it is always true now.